subsys/net/lib/coap_utils/coap_utils.c

```diff
@@ -188,14 +190,16 @@ static void coap_set_response_callback(struct coap_packet *request,
 	reply->reply = reply_cb;
 }
 
-void coap_init(int ip_family)
+void coap_start(int ip_family)
 {
 	proto_family = ip_family;
 
 	fds.events = POLLIN;
 	fds.revents = 0;
 	fds.fd = coap_open_socket();
```

Comment:
will this work?
coap_open_socket depends on is_active but you set is_active after it is called.

Reply:
> will this work?
> coap_open_socket depends on is_active but you set is_active after it is called.

`coap_open_socket` does not depend on `is_active`

Reply:
same as previous - code folding showed coap_open_socket with coap_receive content:)